packages/app-builder/src/routes/_builder+/scenarios+/$scenarioId+/test-run+/$testRunId.tsx

```diff
@@ -75,12 +82,25 @@ export default function TestRun() {
     <Page.Main>
       <Page.Header className="gap-4">
         <Page.BackLink
-          to={getRoute('/scenarios/:scenarioId/test-run', {
+          to={getRoute('/scenarios/:scenarioId/test-run/', {
             scenarioId: fromUUID(currentScenario.id),
           })}
         />
         <p className="line-clamp-2 text-start">{currentScenario.name}</p>
         <TriggerObjectTag>{currentScenario.triggerObjectType}</TriggerObjectTag>
+        <CancelTestRun testRunId={run.id}>
+          <Button
+            variant="secondary"
+            color="red"
+            className="isolate h-10 w-fit"
+            disabled={!hydrated}
```

Comment:
No need, this will be injected at the modal level

Reply:
What are you talking about ? The `testRunId` prop or the `<Button />` ?

Reply:
The `disabled={!hydrated}`